src/Pimple/PimpleContainerAdapter.php

```diff
@@ -48,6 +48,8 @@ public function addServiceConfig(ServiceConfig $config)
     }
 
     /**
+     * @param InflectorConfig $config
```

Comment:
Since the sole purpose of these doc comments is to inform of types when they cannot be determined from the code, I question the value in having the for params which are typehinted. For me, they just feel like an opportunity for misinformation. In our internal code, we only annotate params which don't have typehints. Thoughts?